### ISSUE TITLE
Clarify installation and tests

### ISSUE THREAD

**Issue by pheyvaer** [SolidLabResearch/solid-stream-aggregator#26]
Hi @argahsuknesib 

Can you explain how to install the application? Is it just `npm i`? How do I run it? I see different start commands. How do the tests work? I see some test commands, but it's not clear to me which ones I should run and what the expected outcome is.

It's good practice to have all this written down in the README.

**Comment by argahsuknesib** [comment]
Hello @pheyvaer ,
Thanks for the issue. I have updated the README. Please let me know if anything is unclear. Thanks!

**Comment by pheyvaer** [comment]
@argahsuknesib Thanks! I did expect a PR instead though. And that this issue is only closed once the PR is merged.

**Comment by argahsuknesib** [comment]
ok makes sense, I will make a

**Comment by pheyvaer** [comment]
Every separate issue should have a separate branch. So if that branch fixes this issue then do a pull request and add me as reviewer.

**Comment by argahsuknesib** [comment]
Hello @pheyvaer, can this issue be closed as the PR is merged?

**Comment by pheyvaer** [comment]
Yes indeed. This can happen automatically next time if you put in your commit `Fix #26`. You find more info [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue).

**Comment by argahsuknesib** [comment]
Thanks!
